SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -19,11 +23,26 @@ public ConferenceServiceImpl(ConferenceRepository conferenceRepository) {
 
     @Override
     public List<Conference> listConferences() {
-        return conferenceRepository.findAll();
+        List<Conference> conferences = conferenceRepository.findAll();
+        return conferences;
     }
 
     @Override
-    public Optional<Conference> findConferenceById(Long conferenceId) {
-        return conferenceRepository.findById(conferenceId);
+    public Conference findConferenceById(Long conferenceId) throws ConferenceNotFoundException {
```

Comment:
Why are your exceptions in your method signatures?

Reply:
Should they not be? I think Intelij put them in when we added them along with the imports, I can delete them.